### ISSUE TITLE
Investigate `test_inplace_view_saved_output` failures on Windows

### ISSUE THREAD

**Issue by umanwizard** [pytorch/pytorch#21176]
## 🐛 Bug

Please see details in the description of #21175 

## To Reproduce

1. After #21175 lands, uncomment the line `@unittest.skipIf(IS_WINDOWS, "This test is flaky on Windows")` on `test_inplace_view_saved_output` in `test_autograd.py`
1. Run the test on Windows and observe that it fails. Adding `gc.collect()` does not help.

**Comment by umanwizard** [comment]
Removing the assignment, and the "triaged" tag, since we understand this better now and no longer think it's because of Sam's test.

**Comment by jerryzh168** [comment]
@umanwizard is this resolved?

**Comment by umanwizard** [comment]
@jerryzh168 No.

**Comment by jerryzh168** [comment]
what should we do now? why do you remove "triaged" tag?

**Comment by umanwizard** [comment]
We can add it back, I suppose, since we know the module is "windows".

**Comment by iremyux** [comment]
Hello @umanwizard , are you still having this problem? If yes can you still provide more details on how to reproduce?

**Comment by umanwizard** [comment]
@iremyux I don’t know. I last worked on PyTorch in 2019. This issue was so frustrating I had to change jobs. (Just kidding about the last part).

I think the description is clear, though. Adding a native method called `T` caused undefined behavior making various tests fail on windows. This is why we never added a function to match `ndarray.T` from numpy, but rather used a lowercase `t`.

**Comment by umanwizard** [comment]
Rename the function called `t` to `T` and see what happens. No, I don’t remember how to do this, because I last worked on PyTorch six years and two jobs ago.

**Comment by taras-janea** [comment]
Looked into this on the current main. 
Adding a `T` method to `native_functions.yaml` makes the codegen emit `T.h`, `T_ops.h`, `T_native.h`, etc. under `ATen/ops/`. On Windows these are the same paths as the existing `t.h`, `t_ops.h`, `t_native.h` (NTFS is case-insensitive), so the `t` operator's headers get silently overridden.

Closing as won't fix. If you have any related concerns, please don't hesitate to open a new issue.

**Comment by umanwizard** [comment]
@taras-janea That was not the issue. There was no `t` operator when I opened this issue. I am the one who created the `t` operator, intending it to be named `T`.
